src/content/docs/openai/ChatGPT/chatGPT-case-study-e0402.md

```diff
@@ -0,0 +1,179 @@
+
+ChatGPT now offers both a free version as well as a paid subscription version (ChatGPT Plus). Benefits of the subscription plan includes:
+
+- Faster response times (especially during peak hours)
+- An increased reliability, where downtime and performance issues are less likely
+- Early access to new features, updates and beta-versions
+- A higher usage limit, compared to the free version
+
+Current cost of the subscription version is $20 per month, with a slightly higher cost in the EU (due to taxes) of $23.80.
+
+OpenAI have released an API for accessing the AI models they have developed. On their [official website](https://openai.com/blog/openai-api) they state:
+'_Unlike most AI systems which are designed for one use-case, the API today provides a general-purpose “text in, text out” interface, allowing users to try it on virtually any English language task. You can now request access in order to integrate the API into your product, develop an entirely new application, or help us explore the strengths and limits of this technology._'
+
+The API will return a text completion, from a given text prompt, attempting to match the pattern it was given. It can be "programmed" by providing some examples of what you'd like it to do. The API currently runs models with weights from the GPT-4 family.
+
+### How it works
+
+ChatGPT uses a deep neural network architecture called a Transformer, which was introduced in 2017. Transformers are designed to process sequential data, such as text, and are particularly well-suited to generating text that is coherent and grammatically correct. Since it is trained using a large amount of text data, it learns the statistical patterns and relationships that exist within human language. When a user inputs a prompt, ChatGPT uses this knowledge to generate a response that is relevant.
```

Comment:
I think it will make more sense to talk briefly about prompts. Maybe mention what is a prompt, give some examples of prompts that chatgpt can effectively process with screenshots and establish some typical rules one can follow to make the best use of chatgpt.

Reply:
@cnnrbrn, I believe we can merge this. Another student can extend it by including information on the significance of prompts for maximizing ChatGPT's effectiveness. However, @e0402 can append this to the end of the existing content

```
### Further Research
- Explore the importance of a prompts to the effective use of ChatGpt
```

Reply:
Hi @e0402, we can merge after you add @sulenchy's suggestion.

Reply:
@cnnrbrn I have now added the suggestion.